### PR TITLE
Task homework17

### PR DESCRIPTION
**Pull request by ritabunets**
Homework17 ready to be filled with your comments

### CODE REVIEW

File: Homework17/Tests/UsersTests.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
+﻿using Homework17.Helpers;
+using Homework17.Models.JsonModels;
+using Homework17.Services;
+using NUnit.Framework;
+using System.Net;
```
Comment (yauhenists) [inline Homework17/Tests/UsersTests.cs]: Please move System usings to the first place

File: Homework17/Tests/UsersTests.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        // Test 1: Get list of users and check the number of users.
+        [Test]
+        public void CheckNumberOfUsers()
+        {
+            const int expectedTotalUsers = 12;
+
+            var response = RequestsService.GetUsersList();
+            Assert.Multiple(() =>
+            {
+                Assert.AreEqual(HttpStatusCode.OK, response.StatusCode);
```
Comment (yauhenists) [inline Homework17/Tests/UsersTests.cs]: First 2 asserts absolutely the same. You can leave only one

File: Homework17/Tests/UsersTests.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                Assert.AreEqual(200, (int)response.StatusCode);
+                Assert.AreEqual(expectedTotalUsers, response.ResponseModel.Total);
+            });
+        }
+
+        // Test 2: Get list of users and check if a random user exists.
+        [Test]
+        public void CheckRandomUsers()
```
Comment (yauhenists) [inline Homework17/Tests/UsersTests.cs:28]: Strange test)

File: Homework17/Tests/UsersTests.cs (hunk 2 above)
Comment (yauhenists) [reply Homework17/Tests/UsersTests.cs]: The same below

**Review by yauhenists — CHANGES_REQUESTED** [review]

File: Homework17/Tests/UsersTests.cs (hunk 1 above)
Comment (ritabunets) [reply Homework17/Tests/UsersTests.cs]: oh yeeeees

File: Homework17/Tests/UsersTests.cs (hunk 2 above)
Comment (ritabunets) [reply Homework17/Tests/UsersTests.cs]: yep, it was just to show that i know both ))

File: Homework17/Tests/UsersTests.cs (hunk 3 above)
Comment (ritabunets) [reply Homework17/Tests/UsersTests.cs:28]: it was like my improvisation according to your example "Получить список юзеров, создать рандомный айдишник (нет в списке юзеров), отправить запрос с этим айдишником и проверить ответ – Not Found"))

File: Homework17/Tests/UsersTests.cs (hunk 1 above)
Comment (ritabunets) [reply Homework17/Tests/UsersTests.cs]: i removed it at all

File: Homework17/Tests/UsersTests.cs (hunk 3 above)
Comment (yauhenists) [reply Homework17/Tests/UsersTests.cs:28]: And where the part about "Not Found" in the response?)

**Review by yauhenists — APPROVED** [review]

**Pull request merged**